modules/restarting-installation.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// installing/installing-troubleshooting.adoc
+
+[id="restarting-installation_{context}"]
+= Reinstalling the {product-title} cluster
+
+If you are unable to debug and resolve issues in the failed {product-title} installation, you may need to consider installing a new {product-title} cluster. Before starting the installation process again, you must do a thorough cleanup.
+
+.Procedue
+
+1. Destroy the cluster and remove all the resources associated with the cluster, including the hidden installer state files in the installation directory:
++
+[source, terminal]
+----
+$ ./openshift-install destroy cluster --dir=<installation_directory> <1>
+----
+<1> `installation_directory` is the directory you specified when you ran `./openshift-install create cluster`. This directory contains the {product-title}
+definition files that the installation program creates.
+
+2. xref:../installing/index.adoc#ocp-installation-overview[Install a new {product-title} cluster].
```

Comment:
We still can't put xrefs in modules, unfortunately. Since these steps already exist, I would suggest putting "Follow procedure for Installing a new {product-title} cluster." as step 2. Then in the assembly file, add an additional resource section and write: For details, see xref:....[Install a new {product-title} cluster]. Since this is the last module on the page, the Additional Resources should be perfectly positioned under that step 2. Let me know if I can clarify anything.

Reply:
Thanks for the review. Addressed this comment.